### ISSUE TITLE
visual representation to homepage highlighting regional, thematic, etc components

### ISSUE THREAD

**Issue by 7yl4r** [marinebon/www_marinebon2#34]
The idea is to have a graphic with regions, themes, etc with clickable elements.

I think someone's student is working on this. Who was that?

**Comment by bbest** [comment]
Here are some other icon resources:

* [Freepik](https://www.freepik.com/)
* [FlatIcon](https://www.flaticon.com/)

If we want to do this with [infographiqJS](https://marinebon.org/infographiqJS), then here are some tips for how to proceed with building the illustration (as an SVG) using either:

* [Adobe Illustrator](https://marinebon.org/infographiq/image-creation.html#illustrator) or
* [Inkscape](https://marinebon.org/infographiq/image-creation.html#inkscape)

**Comment by 7yl4r** [comment]
After discussion today I made the following mockup:

![image](https://user-images.githubusercontent.com/1051390/148577738-92c7a4b9-f681-4506-b9b1-d564a3a9ff08.png)

**Comment by 7yl4r** [comment]
[Concept powerpoint from joana](https://docs.google.com/presentation/d/12iwEtwfwglYO-NgSoZz3LXGanUKz8a5o/edit?usp=sharing&ouid=103905248420690900336&rtpof=true&sd=true)

**Comment by 7yl4r** [comment]
Primary considerations:
1. ease of implementation
2. ease of maintainence
3. visual appeal

Overview of options:
1. 2d custom-graphic : infographiq or similar d3-based
2. 3d globe : something webGL-based
3. 2d map : leaflet

--------------------

## Some links: (I will come back and organize these soon)

* https://miniature.earth/demo/european-union.htm
* https://earth.nullschool.net/#current/wind/surface/level/orthographic=-86.84,32.43,335
* https://deck.gl/examples/globe-view/
* https://earth.nullschool.net/
* https://codecanyon.net/item/miniature-earth-interactive-3d-globe-for-javascript/23390670?irgwc=1&clickid=WlX2ShykLxyIUTy3Q3xW0SRsUkGygiSxTyVNwE0&iradid=275988&irpid=1970830&iradtype=ONLINE_TRACKING_LINK&irmptype=mediapartner&mp_value1=&utm_campaign=af_impact_radius_1970830&utm_medium=affiliate&utm_source=impact_radius
* [Gio.js](https://giojs.org/)
* [CesiumJS – Cesium](https://cesium.com/platform/cesiumjs/)
* [5 Awesome 3D Globe Projects To Inspire You 🌍✨ - JavaScript in Plain English](https://javascript.plainenglish.io/5-awesome-3d-globe-projects-to-inspire-you-f9b7e9c5fd80)
* [WebGL Earth - 3D digital globe for web and mobile devices](https://www.webglearth.com/#ll=21.52319,-81.45479;alt=17002870)
* [Globe.GL | globe.gl](https://globe.gl/)
* [WebGL Earth - open source 3D digital globe written in JavaScrip
* [earth :: a global map of wind, weather, and ocean conditions](https://earth.nullschool.net/#current/wind/surface/level/orthographic=-156.17,-9.70,439)
* [deck.gl | GlobeView Example](https://deck.gl/examples/globe-view/)
* [deck.gl](https://deck.gl/)
* [x.knit](http://bwlewis.github.io/rthreejs/globe/index.html)

**Comment by 7yl4r** [comment]
I am still enjoying reading about ocean-centric map projections. Some others from Spilhaus are [here](https://www.jasondavies.com/maps/spilhaus/); they are much more relatable to us land-dwellers. In particular "Shoreline Map XXI" might be a good compromise:

![image](https://user-images.githubusercontent.com/1051390/151067790-8897baac-c128-4e3d-887f-8a5b4da2807a.png)

**Comment by bbest** [comment]
Hey @7yl4r,

Check out this cool interactive globe Brian at Axiom implemented for Sanctuaries and Sound Data Portal:

https://sanctsound.portal.axds.co
![image](https://user-images.githubusercontent.com/2837257/155819196-bd980acd-5c28-4af2-8c58-251d832d54c9.png)

- hover over Channel Islands
  ![image](https://user-images.githubusercontent.com/2837257/155819112-374ba806-c918-4813-8b9f-29684331d6f6.png)
- hover over Florida Keys
  ![image](https://user-images.githubusercontent.com/2837257/155819165-da60b685-7496-4725-baac-aca095e62c17.png)

**Comment by bbest** [comment]
@7yl4r found the above example:

* controls globe with this custom JS: [https://sanctsound.portal.axds.co/app/views/pages/sanctsound/header/data_access.js?v=2022.02.24.23.05](https://sanctsound.portal.axds.co/app/views/pages/sanctsound/header/data_access.js?v=2022.02.24.23.05)
* is based off Chrome Experiment from Google's Data Arts group:
  [dataarts/webgl-globe](https://github.com/dataarts/webgl-globe#data-format): WebGL Globe is a platform for visualizing latitude longitude based information using WebGL

**Comment by 7yl4r** [comment]
revisiting the best of the 3d options in more detail:

tech              | pros           | cons        |     path forward
[miniature earth](https://codecanyon.net/item/miniature-earth-interactive-3d-globe-for-javascript/23390670)        | seems to have everything we want | costs $100  | purchase & build off of [the world news demo](https://miniature.earth/demo/world-news.htm)
webgl globe       | FOSS           | difficulty of custom coding needed is unclear | start from [sanctsound example](https://sanctsound.portal.axds.co/) inspiration
arcgis js         | awesome demo & docs | costs? licensing? | start from [this example](https://www.esri.com/arcgis-blog/products/js-api-arcgis/3d-gis/interactive-3d-globe/)
